src/SchemaRegistry.avro.spec.ts

```diff
@@ -509,6 +509,39 @@ describe('SchemaRegistry', () => {
         ;({ schema } = await schemaRegistry['_getSchema'](registeredSchema.id))
       })
 
+      it('should be able to encode/decode enums schemas', async () => {
```

Comment:
it is a bit confusing how "encode" and decode" are the same test case - usually I would expect them to do the opposite of each other.

Reply:
How should I test the buffer?

Reply:
I don't know - but would have expected a test case for each scenario - one to _encode_ and one to _decode_

Reply:
I understand - but I don't know how to test the buffer - but can prove that the generated schema can encode and decode with a satisfying result.